src/main/java/uk/gov/hmcts/darts/common/entity/CourthouseEntity.java

```diff
@@ -37,4 +42,10 @@ public class CourthouseEntity extends CreatedModifiedBaseEntity {
 
     @OneToMany(mappedBy = "courthouse")
     private List<CourtroomEntity> courtrooms;
+
+    @ManyToMany
```

Comment:
If we need to define a bidirectional ManyToMany relationship here - then I think we should use mappedBy attribute using `SecurityGroupEntity_.COURTHOUSE_ENTITIES`
 So no need to specify the JoinTable on the child side.
e.g. `@ManyToMany(mappedBy = COURTHOUSE_ENTITIES)`

Reply:
Applying above changes caused following errors: 
Caused by: org.hibernate.AnnotationException: Collection 'uk.gov.hmcts.darts.common.entity.CourthouseEntity.securityGroups' is 'mappedBy' a property named 'securityGroupEntities' which does not exist in the target entity 'uk.gov.hmcts.darts.common.entity.SecurityGroupEntity'

Reply:
It should be `SecurityGroupEntity_.COURTHOUSE_ENTITIES`
That generated file has the Java property `courthouseEntities` - so it should be ok.
Think you tried the wrong thing?

Reply:
Please try using `@ManyToMany(mappedBy = SecurityGroupEntity_.COURTHOUSE_ENTITIES)`